static-site/content/resource-listing.yaml

```diff
@@ -62,23 +62,23 @@ coreShowcase:
     img: ncov.png
     filters:
       - ncov
-  - name: Seasonal Flu
+  - name: Seasonal influenza
     img: seasonalinfluenza.png
     filters:
       - seasonal-flu
   - name: Avian Flu
```

Comment:
```suggestion
  - name: Avian Flu
  - name: Avian Influenza
```

Reply:
Fixed in [force-push](https://github.com/nextstrain/nextstrain.org/compare/36af2e91337f13b1a1da138d2dfef5d040b37288..833b1146cfdbb9af1e8eb207a1eb964b76967ae4)